lib/frameit/editor.rb

```diff
@@ -172,18 +172,22 @@ def put_title_into_background(background)
       sum_width = title.width
       sum_width += keyword.width + keyword_padding if keyword
 
-      # Resize the 2 labels if necessary
-      smaller = 1.0 # default
-      ratio = (sum_width + keyword_padding * 2) / image.width.to_f
-      if ratio > 1.0
-        # too large - resizing now
-        smaller = (1.0 / ratio)
-
-        UI.message "Text for image #{self.screenshot.path} is quite long, reducing font size by #{(ratio - 1.0).round(2)}" if $verbose
```

Comment:
Ah, sorry! I had started playing with this before you changed the output to `UI.message` and must have copied the old thing over. Will change it back!